chatty-clownfish/bangazon-sprint-2
# Sw 20 product remove

## Link to Ticket
Closes # [Ticket 20](https://github.com/chatty-clownfish/bangazon-sprint-2/issues/20)

## Description of Proposed Changes

- 

## Steps to Test

Outline the steps to test
- Go to My Products and you should be able to delete a product 

```sh
git fetch --all
git checkout SW-20-product-remove
```


## Impacted Areas in Application

List general components of the application that this PR will affect:

* 

## Mentions @username

Tag users that need to review this code
